packages/augur-core/source/contracts/reporting/ReputationToken.sol

```diff
@@ -35,6 +35,8 @@ contract ReputationToken is VariableSupplyToken, IV2ReputationToken {
         parentUniverse = _parentUniverse;
         warpSync = _augur.lookup("WarpSync");
         legacyRepToken = IERC20(augur.lookup("LegacyReputationToken"));
+        require(warpSync != address(0));
+        require(legacyRepToken != IERC20(0));
         erc1820Registry = IERC1820Registry(_erc1820RegistryAddress);
```

Comment:
Need one for `erc1820Registry` as well, plus the same thing in several other contracts.

Reply:
It enters erc1820Registry on the next line, there is an implicit check (technically, on extcodesize != 0).

Reply:
Good point @epheph.  However, there is still the same issue in multiple other contracts.  Not sure if @nuevoalex would prefer a bunch of small PRs, or a list in GitHub, or to just hunt them down yourself?

Reply:
I'd prefer them being explicitly listed so they can be evaluated and we all see that.